components/StreamField/blocks/section_block/SectionBlock.tsx

```diff
@@ -0,0 +1,15 @@
+import type { SectionBlockProps } from './types';
+import SECTION_BLOCKS from './blocks';
+
+const SectionBlock = ({ title, content }: SectionBlockProps) => (
+    <>
+        <h2>{title}</h2>
+        {content.map((block, i) => {
+            const Block = SECTION_BLOCKS[block.type];
+            // @ts-ignore
```

Comment:
Can we add a comment explaining the issue being ignored? You may need to add an optional parameter of type key  to your block types

Reply:
I've updated this part to avoid having `@ts-ignore`.